benchmarks/TFLite/android-arm64-v8a.cmake

```diff
@@ -467,35 +467,88 @@ iree_benchmark_suite(
     "--task_topology_group_count=4"
 )
 
-# CPU, VMVX, 4-thread, big-core, full-inference
-# VMVX is slow and we're not optimizing perf yet. Leaving in a single max-thread
-# benchmark because it's useful to keep an eye on and helps disambiguate where a
-# performance change may be coming from (e.g. if it's in vmvx as well, it's
-# probably not a codegen issue).
+# VMVX benchmarks with ukernels for various ARM64 ISA variants.
+# Single-thread only while we have enough single-thread performance problems
```

Comment:
if you use more than 1 core it'll go faster :)

Reply:
but the new development here is i actually care about the metric as it's going to be tracking my work over the next month, so i want it to be stable and to have a simple interpretation. both traits are helped by sticking to 1 thread for now.

Reply:
How much does this impact overall latency of the benchmark pipeline? Contrasting [your run](https://buildkite.com/iree/iree-benchmark-android/builds/4244) with a [couple](https://buildkite.com/iree/iree-benchmark-android/builds/4296)[runs](https://buildkite.com/iree/iree-benchmark-android/builds/4297) from mainline, I'm seeing what looks like an extra 15 minutes (+50%) for the benchmarks on the Pixel phones. I think that's too much of a latency hit